### ISSUE TITLE
IEPlugin removed from upcoming OpenVINO versions

### ISSUE THREAD

**Issue by benhoff** [opencv/cvat#1830]
Currently the Auto Annotation app uses the class `IEPlugin` from openvino. See here: https://github.com/opencv/cvat/blob/76280be4ad4c6143533b6a3960a429d33e7f10a8/cvat/apps/auto_annotation/inference_engine.py#L5

In upcoming versions of OpenVINO, this class has been removed. See here for more information:
https://github.com/openvinotoolkit/openvino/pull/920

This will cause new installations of CVAT to break when using newer versions of OpenVINO with an error of: 
`ImportError: cannot import name 'IEPlugin' from 'openvino.inference_engine'`

I believe the core of this is already handled in this commit: 14084435bcc856bce4bbc0c0a2f2506233d58ea0

Probably need to put a try/catch around IEPlugin to degrade gracefully.

**Comment by benhoff** [comment]
Related to this:

```
cvat/apps/auto_annotation/model_loader.py:28: DeprecationWarning: 'inputs' property of IENetwork class is deprecated. To access DataPtrs user need to use 'input_data' property of InputInfoPtr objects which can be accessed by 'input_info' property.
  iter_inputs = iter(network.inputs
```

**Comment by nmanovic** [comment]
Should be fixed by #1767
